hotsite/mainpage/style/prototype.css

```diff
@@ -0,0 +1,84 @@
+:root {
+
+    --background-color: black;
+    --text-color: #9c8657;
+}
+
+* {
+
+    font-family: "MedievalSharp", cursive;
+
+}
+
+body {
+    text-align: center;
+
+    background-color: black;
+}
+
+header {
+
+    background-color: var(--background-color);
+
+    color: var(--text-color);
+
+}
+
+.Logo {
+
+    width: 462px;
+
+}
+
+.page-title {
+
+    font-weight: 400;
+}
+
+.button {
```

Comment:
try to avoid too much breaking lines in you css page. same reason of html breaking lines comment.

Example of good approach:
```css
.button {
    border: #9c8657 solid 1px;
    display: inline-block;
    color: var(--text-color);
}
```